src/webhook/pkg/mutating_webhook/remove-resourcespec.go

```diff
@@ -1,17 +1,26 @@
 package mutating_webhook
 
 import (
+	"log"
 	"os"
 
+	"github.com/confidential-containers/cloud-api-adaptor/src/webhook/pkg/utils"
+
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
 const (
 	RUNTIME_CLASS_NAME_DEFAULT       = "kata-remote"
 	POD_VM_EXTENDED_RESOURCE_DEFAULT = "kata.peerpods.io/vm"
+	PEERPODS_CPU_ANNOTATION          = "io.katacontainers.config.hypervisor.default_vcpus"
+	PEERPODS_MEMORY_ANNOTATION       = "io.katacontainers.config.hypervisor.default_memory"
+	GPU_RESOURCE_NAME                = "nvidia.com/gpu"
+	PEERPODS_GPU_ANNOTATION          = "io.katacontainers.config.hypervisor.default_gpus"
```

Comment:
Where is this annotation implemented? I can't find it in the runtime (either main or CCv0)

Reply:
This is hallucination :-( ..
I wanted to just add a custom annotation to indicate the gpu resources in the original request for potential use in cloud-api-adaptor code later on. Forgot to add the right commit.